src/main/java/com/google/cloud/tools/libraries/libraries.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0">
+<!-- Warning: non-normative. Probably not even correct. -->
+<apis>
+
+  <api name="BigQuery API v2"
+      definition="https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest"
+      documentation="https://cloud.google.com/bigquery/docs/reference/rest/v2/"
```

Comment:
Why this page?  As compared to https://cloud.google.com/bigquery/docs/reference/  or the `documentationLink` attribute from the `definition` URL